internal/system/agent/init.go

```diff
@@ -32,8 +32,9 @@ var Configuration *interfaces.ConfigurationStruct
 var Conf = &interfaces.ConfigurationStruct{}
 var clients map[string]general.GeneralClient
 
-// executorClient is the empty interface so that we may type cast it
-// to whatever operation we need it to do at runtime
+// Note that executorClient is the empty interface so that we may type-cast it
+// to whatever operation we need it to do at runtime. Also, executorClient is
```

Comment:
Just remove the sentence `Also, executorClient is no longer set on initialization.` from this comment and this is okay